scripts/installscripts/buster-install-default.sh

```diff
@@ -523,15 +522,46 @@ config_spotify() {
             read -rp "Type your Spotify password: " SPOTIpass
             read -rp "Type your client_id: " SPOTIclientid
             read -rp "Type your client_secret: " SPOTIclientsecret
+            clear
+            echo "#####################################################
+#
+# Swapfile creation
+#
+# With the current way Spotify support is build, a lot of RAM needs
+# to be available in order for the required components to build
+# correctly. This includes even recent Raspberry Pi versions.
+# In general: If less than 2 GB RAM is available in your system, the
+# build of gst-plugin-spotify will most certainly fail.
+#
+# This script can detect if a swapfile is required and have one
```

Comment:
Is the question below, before or after the necessity for a swap file is checked?